style.css

```diff
@@ -357,6 +383,29 @@ body {
     gap: 0.75em;
   }
 
+  .popup {
+    position: fixed;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    overflow: auto;
+    background: rgba(0, 0, 0, 0.7);
+    padding-left: 16px;
+    padding-right: 16px;
+    padding-bottom: 39px;
+    padding-top: 60px;
+    z-index: 2;
+    visibility: hidden;
+  }
+
+  .popup_content {
+    display: flex;
+    flex-direction: column;
+    background: #fff;
+    border-radius: 2px;
+    padding: 16px;
+  }
```

Comment:
Well done on implementing the popup in the desktop version! To **bring it even closer to the [provided template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=6Kf7cezM539HxgRb-0)**, I have a few styling suggestions:

- [ ] Please verify that you are using the correct size of the image in your popup. Additionally, **consider replacing it with the image provided in the template for better accuracy**. 👌

- [ ] Ensure that your buttons are aligned correctly, following the design specifications in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=6Kf7cezM539HxgRb-0). They should be positioned at the bottom left. **This adjustment will enhance the overall professionalism and craftsmanship of your popup.** 😉

  _By making these small refinements, you'll achieve a more faithful representation of the template and create a polished and visually appealing desktop popup. Keep up the great work!_ 🎯

  ### Your design:
  ![image](https://github.com/Kanyorok/Portfolio_Setup/assets/107952724/40e817cf-4f9b-4f2b-85a9-01889710be7e)
  ### Figma design:
  ![image](https://github.com/Kanyorok/Portfolio_Setup/assets/107952724/fca0837d-8c71-4751-8a60-d22c4ad47728)
  
    **_Just a friendly reminder that adhering to the template is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md). Thanks for considering these suggestions, and keep up the great work!_**
![image](https://github.com/Alibaba2023/My_Portfolio/assets/107952724/c24da992-2083-4760-b22b-7f2bf13e775a)